### ISSUE TITLE
Ombi isn't doing a full sync with the Plex server

### ISSUE THREAD

**Issue by goober2186** [tidusjar/Ombi#2744]
#### Ombi build Version:

V 3.0.4144

#### Update Branch:

Development

#### Media Sever:

Plex

#### Media Server Version:

1.14.1.5488

#### Operating System:

Ubuntu 16.04

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```

2019-01-19 01:13:45.713 +00:00 [Error] Could not test Plex
System.ObjectDisposedException: Cannot access a disposed object. A common cause of this error is disposing a context that was resolved from dependency injection and then later trying to use the same context instance elsewhere in your application. This may occur if you are calling Dispose() on the context, or wrapping the context in a using statement. If you are using dependency injection, you should let the dependency injection container take care of disposing context instances.
Object name: 'SettingsContext'.
   at Microsoft.EntityFrameworkCore.DbContext.CheckDisposed()
   at Microsoft.EntityFrameworkCore.DbContext.get_DbContextDependencies()
   at Microsoft.EntityFrameworkCore.DbContext.Microsoft.EntityFrameworkCore.Internal.IDbContextDependencies.get_QueryProvider()
   at Microsoft.EntityFrameworkCore.Query.ExpressionVisitors.Internal.ParameterExtractingExpressionVisitor..ctor(IEvaluatableExpressionFilter evaluatableExpressionFilter, IParameterValues parameterValues, IDiagnosticsLogger`1 logger, DbContext context, Boolean parameterize, Boolean generateContextAccessors)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryModelGenerator.ExtractParameters(IDiagnosticsLogger`1 logger, Expression query, IParameterValues parameterValues, Boolean parameterize, Boolean generateContextAccessors)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.ExecuteAsync[TResult](Expression query, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.EntityQueryProvider.ExecuteAsync[TResult](Expression expression, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ExecuteAsync[TSource,TResult](MethodInfo operatorMethodInfo, IQueryable`1 source, Expression expression, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ExecuteAsync[TSource,TResult](MethodInfo operatorMethodInfo, IQueryable`1 source, LambdaExpression expression, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.FirstOrDefaultAsync[TSource](IQueryable`1 source, Expression`1 predicate, CancellationToken cancellationToken)
   at Ombi.Store.Repository.SettingsJsonRepository.GetAsync(String settingsName)
   at Ombi.Settings.Settings.SettingsService`1.<GetSettingsAsync>b__11_0() in C:\projects\requestplex\src\Ombi.Settings\Settings\SettingsService.cs:line 51
   at Ombi.Helpers.CacheService.GetOrAdd[T](String cacheKey, Func`1 factory, DateTime absoluteExpiration)
   at Ombi.Settings.Settings.SettingsService`1.GetSettingsAsync()
   at Ombi.Api.OmbiHttpClient.<GetHandler>b__8_0() in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 86
   at Ombi.Helpers.CacheService.GetOrAdd[T](String cacheKey, Func`1 factory, DateTime absoluteExpiration)
   at Ombi.Api.OmbiHttpClient.GetHandler() in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 86
   at Ombi.Api.OmbiHttpClient.Setup() in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 77
   at Ombi.Api.OmbiHttpClient.SendAsync(HttpRequestMessage request) in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 61
   at Ombi.Api.Api.Request[T](Request request) in C:\projects\requestplex\src\Ombi.Api\Api.cs:line 38
   at Ombi.Api.Plex.PlexApi.GetStatus(String authToken, String uri) in C:\projects\requestplex\src\Ombi.Api.Plex\PlexApi.cs:line 97
   at Ombi.Controllers.External.TesterController.Plex(PlexServers settings) in C:\projects\requestplex\src\Ombi\Controllers\External\TesterController.cs:line 246


```

#### Problem Description:

Ombi isn't doing a full sync with the Plex server. My Ombi.db PlexServerContent is blank. I've tried stable and dev even installing fresh  on a different virtual machine. I just can't get this to work at all.

#### Reproduction Steps:
I don't know what I might be doing wrong or what is causing this issue.

**Comment by tidusjar** [comment]
Take a look in the OmbiExternal.db

This is where we store any content that is external to Ombi now.

So is it not Sync'ing at all? Or is some content but not all?

**Comment by goober2186** [comment]
I actually ended up switching to running on windows and it's working fine there, but I still have it running in ubuntu as well just for this. It does look like everything was being populated in OmbiExternal.db. It just wasn't showing anything as available. Maybe I wasn't giving it enough time, but I did give it overnight and things weren't showing as available.  My issue is fixed by running on windows though, if you want more info I can give it, or we can just close this. I appreciate the response and keep up the great work on Ombi!
